README.md

```diff
@@ -1,23 +1,44 @@
 # website-public
 
-The main C4Coin Public facing website
+The main C4Coin Public facing website, providing sales, marketing, and public relations information.
 
 * `develop` — [insert CircleCI badge]
 * `master` — [insert CircleCI badge]
 
 ## Functional Requirements
 
-_add details_
+1. Provide a curriculum (funnel) for understanding the C4Coin project
+2. Provide an interactive/live development roadmap
+
+  - Represents progress of section 4.5.3 of the whitepaper
+  - Five Milestones
+
+3. Display C4Coin related press and press releases
+4. Drive email list sign ups (to be clarified with PR team)
+
+  - via CRM or custom built system?
+
+5. Drive social media followers (to be clarified with PR team)
+6. Content Management System (CMS) integration
+
+  Q:  Will the CMS control only the blog, or being able to reorganise the entire site?
+
+7. The C4Coin team
+8. Contact information
+9. Legal information
+
+Comments: This site is relatively simple because there are no public user accounts, only internal.
 
 ## Development
 
-_add details_
+The site will be built using `react-create-app` as a base and will conform to the general coding standards common to all of the C4Coin sites.
```

Comment:
@smoliver Also wondering if we should include a reference to the layout you were thinking here: https://hackernoon.com/fractal-a-react-app-structure-for-infinite-scale-4dab943092af